### DISCUSSION TITLE
Light group discovery

### DISCUSSION THREAD

**Discussion by mfregonassi** [tsightler/ring-mqtt#579] [General Discussion]
Hi, I have installed a floodlight pro camera and it gives me the option of associating its light with a new light group in the Ring app.

I wonder if the light group should be created as device during device discovery (it currently doesn't for me, not sure if it's intended) so I could use the light group toggle instead of the floodlight light toggle.

**Comment by tsightler** [comment]
Light groups are supported for Ring Smart Lighting, and should be discovered automatically.  For cameras, even though Ring uses the same terminology of a "Light group", I don't believe it is the same type of entity and thus is not supported.  Personally, I think you are better off grouping lights in HA anyway.
